packages/block-editor/src/components/off-canvas-editor/index.js

```diff
@@ -92,7 +92,8 @@ function OffCanvasEditor(
 
 	const [ expandedState, setExpandedState ] = useReducer( expanded, {} );
 
-	const { ref: dropZoneRef, target: blockDropTarget } = useListViewDropZone();
+	const { ref: dropZoneRef, target: blockDropTarget } =
```

Comment:
Can we make `offcanvas` a prop that we pass to the component, so that when we combine this with the List View we already have the tools we need?